File: cs102/Week 7/Week 7 - Web/lab2.c
#include <stdio.h>
int main (int argc, char** argv) {
    fprintf(stdout,"Content-type: text/html\n\n");
    fprintf(stdout,"<TITLE>Lab 2</TITLE>\n");
    fprintf(stdout,"<h1>Lab 2</h1>\n");
    fprintf(stdout,"<html>\n");
    fprintf(stdout,"<body>\n");
    for (int x = 0; x<100; x=x+1) {
        fprintf(stdout,"<p>%d</p>\n", x+1);
    }
    fprintf(stdout,"</html>\n");
    fprintf(stdout,"</body>\n");
    return 0;
}  

File: cs102/Week 7/Week 7 - Web/lab2.py
#!/usr/bin/python
print( "Content-type: text/html\n\n" )
print("<TITLE>Lab 2</TITLE>")
print("<h1>Lab 2</h1>")
print( """
        <html>
        <body>
""" )
for x in range(100):
    print("<p>%s</p>" % (x+1))
print( """
        </body>
        </html>
""" )
